examples/01-Detailed-Examples/07-mesh-external-layer.py

```diff
@@ -0,0 +1,114 @@
+"""
+.. _ref_mesh_external_layer_example:
+
+Extract Data and Plot on Mesh External Layer
+============================================
+In this example, post-processing on a mesh external layer for a static analysis is displayed.
+This feature is available for all types of MechanicalSimulation and allows you to reduce the size
+of the mesh and of the extracted data to improve processing performances.
+Larger stress and strains being usually located on the skin of a model,
+computing results on the external layer gives equivalent maximum values in most cases.
```

Comment:
```suggestion
In this example, post-processing is displayed on a mesh external layer for a static analysis.
This feature is available for all types of Mechanical simulation, and allows you to reduce the size
of the mesh and of the extracted data to improve processing performance.
Since larger stress and strains are usually located on the skin of a model,
computing the results on the external layer provides equivalent maximum values in most cases.
```